Source/ablastr/constant.H

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2019-2022 Andrew Myers, Luca Fedeli, Maxence Thevenet,
+ *                     Weiqun Zhang, Axel Huebl
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef ABLASTR_CONSTANT_H_
+#define ABLASTR_CONSTANT_H_
+
+#include <AMReX_REAL.H>
+
+
+/** Numerical compile-time constants */
+namespace ablastr::constant
+{
+    /** Mathematical constants */
+    namespace math
+    {
+        using namespace amrex::literals;
+
+        //! ratio of a circle's circumference to its diameter
+        static constexpr amrex::Real pi = 3.14159265358979323846_rt;
+
+        //! https://tauday.com/tau-manifesto
+        static constexpr amrex::Real tau = 2.0 * 3.14159265358979323846_rt;
```

Comment:
Thanks for the link to the tau-manifesto !

I am wondering if writing something like that could be better for readability...
```suggestion
        static constexpr amrex::Real tau = 2.0 * pi;
```